WebApplication/ClientApp/src/components/forgePdfViewExtension.js

```diff
@@ -0,0 +1,102 @@
+// *******************************************
+// My Color Extension
+// *******************************************
+
+const Autodesk = window.Autodesk;
+
+function ForgePdfViewExtension(viewer, options) {
+    this.initialized = false;
+    Autodesk.Viewing.Extension.call(this, viewer, options);
+}
+
+ForgePdfViewExtension.prototype = Object.create(Autodesk.Viewing.Extension.prototype);
+ForgePdfViewExtension.prototype.constructor = ForgePdfViewExtension;
+
+ForgePdfViewExtension.prototype.load = function () {
+
+    if (this.viewer.toolbar) {
+        // Toolbar is already available, create the UI
+        this.createUI();
+    } else {
+        // Toolbar hasn't been created yet, wait until we get notification of its creation
+        this.onToolbarCreatedBinded = this.onToolbarCreated.bind(this);
+        this.viewer.addEventListener(Autodesk.Viewing.TOOLBAR_CREATED_EVENT, this.onToolbarCreatedBinded);
+    }
+
+    return true;
+};
+
+ForgePdfViewExtension.prototype.onToolbarCreated = function () {
+    this.viewer.removeEventListener(Autodesk.Viewing.TOOLBAR_CREATED_EVENT, this.onToolbarCreatedBinded);
+    this.onToolbarCreatedBinded = null;
+    this.createUI();
+};
+
+ForgePdfViewExtension.prototype.createUI = function() {
+    if (this.initialized === true)
+        return;
+
+    this.initialized = true;
+    const viewer = this.viewer;
+    const urn = viewer.getState().seedURN;
+    const numPages = viewer.model.loader.pdf.numPages;
+    let actualPage = 1;
```

Comment:
To be sure: index of the first page is `1`, not `0`?

Reply:
yes, default pageNumber is 1